FreeRTOS/Test/Target/boards/pico/tests/smp/multiple_tasks_running/main.c

```diff
@@ -0,0 +1,146 @@
+/*

```

Comment:
This file needs to be refactored into 2 files - 

1. `main.c` - This file should be put in `FreeRTOS/Test/Target/boards/pico` and should contain common code applicable to all the tests.
2. `multiple_task_running_test_runner.c` - This file should have the following content:

```c
static void prvTestRunnerTask( void * pvParameters )
{
    ( void ) pvParameters;

    /* Run test case. */
    vRunMultipleTasksRunningTest();

    vTaskDelete( NULL );
}

void vRunTest( void )
{
    xTaskCreate( prvTestRunnerTask,
                 "testRunner",
                 configMINIMAL_STACK_SIZE,
                 NULL,
                 configMAX_PRIORITIES - 1,
                 NULL );
}
```